package.json

```diff
@@ -144,20 +145,22 @@
       "packages/fluentui/*"
     ],
     "nohoist": [
-      "@fluentui/babel-make-styles/tslib",
-      "@fluentui/babel-make-styles/typescript",
-      "@fluentui/make-styles/stylis",
       "@fluentui/make-styles/@types/stylis",
+      "@fluentui/make-styles/stylis",
       "@fluentui/react-bindings/stylis",
       "@fluentui/react-northstar-fela-renderer/stylis",
-      "packages/web-components/webpack",
+      "@fluentui/web-components/ts-node",
+      "@fluentui/web-components/ts-node/**",
+      "packages/web-components/**/typescript",
```

Comment:
Can you also update the rest of these to use `@fluentui/web-components` not the path (and sort them)?